index.html

```diff
@@ -230,15 +237,24 @@ <h2 id="contact-me">Contact me </h2>
                                 required></textarea>
                         </li>
                         <li class="input-data">
-                            <button type="submit" class="button">Get In Touch</button>
+                            <button type="submit" id="btn" class="button">Get In Touch</button>
                         </li>
                     </ul>
                 </form>
             </section>
             <hr class="footer-line">
         </footer>
-
     </div>
+
+    <script>
+        function openNavbar() {
+            document.getElementById('mobileNav').style.height = '100%';
+        }
+
+        function closeNavbar() {
+            document.getElementById('mobileNav').style.height = '0%';
+        }
+    </script>
```

Comment:
- [ ] The best practice is to put JavaScript code in its own file and include the file in the HTML. Thus, kindly create new file and add JavaScript code in that file. [reference](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
![JS-best-practice](https://user-images.githubusercontent.com/44057863/168735540-2cd74bbd-72cb-4638-829d-e06b6471b93e.png)